.eslintrc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:jest/recommended',
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'prettier',
+    'prettier/@typescript-eslint',
+  ],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 12,
+    sourceType: 'module',
+  },
+  plugins: ['jest', 'react', '@typescript-eslint'],
+  rules: {},
```

Comment:
Apart from this empty ruleset, all of this looks fine to me.

Reply:
(FWIW, if you have bugs with error on indentation, and then prettier "not" doing it right, manually add

```
"indent": 0,
"@typescript-eslint": 0
```